os/include/tinyara/usb/usbhost.h

```diff
@@ -251,7 +262,7 @@
  *
  ************************************************************************************/
 
-#define DRVR_EP0CONFIGURE(drvr, funcaddr, mps) ((drvr)->ep0configure(drvr, funcaddr, mps))
+#define DRVR_EP0CONFIGURE(drvr,ep0,funcaddr,speed,mps) ((drvr)->ep0configure(drvr,ep0,funcaddr,speed,mps))
```

Comment:
same as above, Please check for space after comma at all places in this file.

Reply:
Done for all the occurrences